examples/sequence_tagging/conll_reader.py

```diff
@@ -0,0 +1,266 @@
+# Copyright 2019 The Texar Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Utilities for pre-processing and iterating over the CoNLL 2003 data.
+"""
+
+import re
+
+from collections import defaultdict
+
+import numpy as np
+
+import texar.torch as tx
+
+# pylint: disable=redefined-outer-name, unused-variable
```

Comment:
Why are these necessary? I can imagine `unused-variable` being required for the constants, but why `redefined-outer-name`? It's also better practice to add a corresponding `pylint: enable` comment after where it's no longer required.